### PR TITLE
Notification controller

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1088

### Change description ###
NotificationController along with integration test

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/notificationservice/controller/NotificationController.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    public NotificationController(
+        NotificationService notificationService,
+        AuthService authService
+    ) {
+        this.notificationService = notificationService;
+        this.authService = authService;
+    }
+
+    @GetMapping()
+    public List<NotificationResponse> getNotifications(
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/notificationservice/controller/NotificationController.java]: in all other projects we always return an object, never an array

File: src/integrationTest/java/uk/gov/hmcts/reform/notificationservice/controller/NotificationControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        mockMvc = webAppContextSetup(wac).build();
+    }
+
+    @Test
+    void should_get_notifications_by_file_name_and_service() throws Exception {
+        final String fileName = "zip_file_name.zip";
+        final String auth = "auth";
+        final String service = "service";
+
+        Calendar cal   = Calendar.getInstance();
```
Comment (rkondratowicz) [inline src/integrationTest/java/uk/gov/hmcts/reform/notificationservice/controller/NotificationControllerTest.java]: formatting

File: src/main/java/uk/gov/hmcts/reform/notificationservice/controller/NotificationController.java (hunk 1 above)
Comment (apereverzin) [reply src/main/java/uk/gov/hmcts/reform/notificationservice/controller/NotificationController.java]: fixed

File: src/integrationTest/java/uk/gov/hmcts/reform/notificationservice/controller/NotificationControllerTest.java (hunk 2 above)
Comment (apereverzin) [reply src/integrationTest/java/uk/gov/hmcts/reform/notificationservice/controller/NotificationControllerTest.java]: fixed

**Pull request merged**